package.json

```diff
@@ -152,7 +152,7 @@
     "history": "4.10.1",
     "hs-airdrop": "0.10.0",
     "hs-client": "0.0.11",
-    "hsd": "https://github.com/handshake-org/hsd#v4.0.1",
+    "hsd": "rithvikvibhu/hsd#multisig-generatenonce",
```

Comment:
Is this temporary?

Reply:
Yes, this depends on 2 hsd PRs:
- batching: merged into master, but not part of v4.0.2
- genereatenonce: not yet merged

See first To Do point, what do you think?

Reply:
rebased so now it uses hsd v5.0.0-rc.1 which includes both.